src/main/kotlin/org/rust/lang/core/psi/RsFile.kt

```diff
@@ -78,7 +78,7 @@ class RsFile(
             val originalFile = originalFile
             if (originalFile != this) return (originalFile as? RsFile)?.cachedData ?: EMPTY_CACHED_DATA
 
-            val state = project.macroExpansionManager.expansionState
+            val state = project.macroExpansionManagerIfCreated?.expansionState
```

Comment:
When is it possible that `project.macroExpansionManagerIfCreated` is null here?

Reply:
When you have a Rust file in the project and no attached cargo projects and even don't open any Rust file, this code can be invoked during determining an icon for the file (it's different for crate root and module) in the project structure or search by filename